modules/domManipulation.js

```diff
@@ -0,0 +1,65 @@
+import BookCollection from './bookCollection.js';
+
+export default function displayBooks() {
+  const newBooksCollection = new BookCollection();
+  const div = document.querySelector('#bookListContainer');
+  const ul = document.querySelector('#ulBookList');
+  const addForm = document.getElementById('addBookForm');
```

Comment:
- [ ] Same here, please use the `ES6` arrow function syntax instead of regular syntax.